src/rf/apps/core/enums.py

```diff
@@ -25,6 +27,7 @@
     (STATUS_CREATED, 'Created'),
     (STATUS_UPLOADED, 'Uploaded'),
     (STATUS_VALIDATED, 'Validated'),
+    (STATUS_INVALID, 'Invalid'),
```

Comment:
I just noticed we are reusing these statuses for layer and layer image which is a little confusing. I suggest we rename `STATUS_VALIDATED` to `STATUS_VALID` for layer image.